skbio/metadata/intersection.pyx

```diff
@@ -0,0 +1,513 @@
+"""
```

Comment:
Is this just a copy of their file? Generally you'd want to check with the developers of the file first (not sure if you've done that here), and then possibly put a note in our copy of the file. You also would need to add a licence to [``scikit-bio/licenses``](https://github.com/biocore/scikit-bio/tree/master/licenses). We should always be discussing these kind of additions beforehand (probably at the RFC stage, when we're discussing any new dependencies) to make sure that all our bases are covered.